application/models/Participant.php

```diff
@@ -996,43 +996,54 @@ public function deleteParticipantTokenAnswer($sParticipantsIDs)
                 ->where('participant_id = :row')
                 ->bindParam(":row", $row, PDO::PARAM_INT)
                 ->queryAll();
-
+		
+			
             foreach ($tokens as $key => $surveyLink) {
```

Comment:
Maybe all of this can be moved to a [afterDelete](https://www.yiiframework.com/doc/api/1.1/CActiveRecord#afterDelete-detail) method :)